CHANGELOG.md

```diff
@@ -38,7 +38,9 @@ Format:
 ### Security
 - 
 -->
-<!-- ## [Unreleased] -->
+## [Unreleased]
+### Changed
+- Allow devs to test list of interviews on their server. See https://github.com/SuffolkLITLab/ALKiln/issues/600.
```

Comment:
I would say that this lets you test installed interviews, right? Nothing in this PR is specific to getting the full list and testing it.